### ISSUE TITLE
git2 dll is built for Windows 7 and higher

### ISSUE THREAD

**Issue by cypok** [excelsior-oss/far-git-autocomplete#6]
С [форума](http://forum.farmanager.com/viewtopic.php?p=141943&sid=2fd0439322b003c568a9ac9534d9a79f#p141943):
> Есть 2е новость хорошая и плохая!
> 1я Сам Плагин работает! Вызов конфига.. (XP SP3)..
> А дале плохая..
> 2я Он не работает так-как "git2_32.dll" собрана для Win7!..

Надо разобраться и пересобрать.

**Comment by cypok** [comment]
Из соседнего обсуждения #5:

> Да, судя по комментариям в обсуждении есть повод и пересобрать плагин (тут я бы воспользовался или VC2010, или GCC поскольку уже приходилось встречать случаи когда и vc1х0_xp тулкит, и записи в мэйке или солюшене указывавшие на необходимость собрать бинарник для ХР студия игнорирует и в импортах находишь вызовы для виста и более старших систем, и сразу ВОМ вставить чтобы исключить "всплытие" этой уже устранённой нами ошибки в принципе - лучше перестраховаться, чем после удивляться. :)

**Comment by cypok** [comment]
Надо заметить, что [начиная с версии 0.21.0 (июнь 2014)](https://github.com/libgit2/libgit2/releases/tag/v0.21.0) libgit2 не поддерживает Windows XP.

**Comment by VictorVG** [comment]
Да, не самое приятное занятие - подбирать по MSDN замену функциям. А если кроме имён там сам вызов надо менять? Лишняя проверка чтобы чего не вылезло....
